pages/trips/[id]/index.js

```diff
@@ -1,10 +1,11 @@
 import ConfirmDelete from "@/components/ConfirmDelete";
 import Image from "next/image";
+import ItemList from "@/components/ItemList";
```

Comment:
I think this import is not needed here, correct? As ItemList is used in PackingList? :)

Reply:
@e-rmkt thank you for your very nice comment. And also for your review of cause :-D.
So unneeded imports and also console.logs are removed.